neqo-http3/src/connection_client.rs

```diff
@@ -292,15 +296,21 @@ impl Http3Client {
         }
         final_headers.extend_from_slice(headers);
 
+        let mut send_message = SendMessage::new(
+            id,
+            self.base_handler.qpack_encoder.clone(),
+            Box::new(self.events.clone()),
+        );
+
+        // SendMessage implements http_stream so ith will not panic.
+        send_message
+            .http_stream()
+            .unwrap()
```

Comment:
Do you need these lines?  At a minimum you should be able to do `(send_message as HttpStream).send_headers()` as the thing you have is fine.

Reply:
This does not work. I have tried it.